templates/github/.github/workflows/nightly.yml.j2

```diff
@@ -74,7 +73,7 @@ jobs:
 
       {{ setup_python() | indent(6) }}
 
-      {{ install_python_deps("requests") | indent(6) }}
+      {{ install_python_deps(["requests"]) | indent(6) }}
```

Comment:
I would also install this requirement inside the caller ".github/workflows/scripts/publish_docs.sh". Do you think it is worth it to revisit all installing dependencies and move them to one place?

Reply:
I intended to stop installing anything from scripts, because i consider that to be dangerous if you try to run any of these locally.

Reply:
Gotcha, then it makes sense to remove pip calls from the scripts, in another PR.

Reply:
And i dind't even check with "git grep" ... XD